src/ubx-message.toit

```diff
@@ -660,26 +696,108 @@ class CfgMsg extends Message:
   static MAX-PROTVER/string := ""
 
   /**
-  Constructs a configuration message.
+  Constructs a blank configuration-rate message.
+
+  Examine/configure this message using the supplied getters and setters.  If
+    sent to the device it may overwrite the rates for all ports for the
+    specified message type.
```

Comment:
Is this comment to emphasize that *all* ports are affected?
Why "may"?
Why "overwrite" and not "set"?

```suggestion
  When sent to the device it may overwrite the rates for all ports for the
    specified message type.
```

Also, I would probably remove the "{if|when} sent".